packages/test/test-end-to-end-tests/src/test/sharedInterval.spec.ts

```diff
@@ -657,6 +657,7 @@ describeFullCompat("SharedInterval", (getTestObjectProvider) => {
                 if (typeof(intervals1.changeProperties) === "function" &&
                     typeof(intervals2.changeProperties) === "function") {
                     intervals1.changeProperties(id1, { prop1: "prop1" });
+                    await provider.opProcessingController.processOutgoing();
```

Comment:
Is this currently causing test result instability?

Reply:
Yes. It's similar to the change Curtis made a couple of days ago. Now I need to apply it to the calls to changeProperties that I added with my latest change.